fec-icons/icon-template.scss

```diff
@@ -0,0 +1,31 @@
+// NOTE: Do note edit _icon-variables.scss directly
+// It is generated by Gulp from fec-icons/icon-template.scss
```

Comment:
Aah I misunderstood and thought you were talking about this file, which generates the final file. I see your point, but then it would mean that every app that included this library would have to generate the icon file itself, which doesn't seem like a good idea since that step is required for the icons to show up. 

All that to say, I _think_ what we have now is best, though to your point, we'll pin to specific versions of the plugins used to generate the strings. What do you think?